app/src/App.tsx

```diff
@@ -69,18 +78,25 @@ export const App = (): React.ReactElement => {
         <meta charSet='utf-8' />
         <title>Divergent Console</title>
       </Helmet>
-      <Router>
-        <Route path='/' page={EmptyPage} />
-        <Route path='/login' page={LoginPage} />
-        <Route path='/register' page={RegistrationPage} />
-        <Route path='/forgot-password' page={ForgotPasswordPage} />
-        <Route path='/domains' page={DomainsPage} />
-        <Route path='/domains/create' page={DomainCreatePage} />
-        <Route path='/domains/:domainId' page={DomainPage} />
-        <Route path='/domains/:domainId/links/:linkId' page={LinkPage} />
-        <Route path='/domains/:domainId/links/create' page={CreateLinkPage} />
-        <Route default={true} page={NotFoundPage} />
-      </Router>
+      <Stack direction={Direction.Vertical} isFullWidth={true} isFullHeight={true} isScrollableVertically={false} contentAlignment={Alignment.Start}>
+        <NavBar />
+        <Stack.Item growthFactor={1} shrinkFactor={1}>
+          <Box isScrollableVertically={true}>
```

Comment:
this is kinda annoying, will try think of a better structure for this. I think it may need to be a new layout entirely